torch/csrc/lazy/ts_backend/ts_eager_fallback.cpp

```diff
@@ -153,6 +153,12 @@ bool force_eager_fallback(c10::Symbol op) {
       return true;
     }
   }
+  if(op == at::aten::nonzero){
+    // When symbolic shape mode is not enabled, the nonzero shape function
+    // returns an incorrect result.
+    return !symbolicShapeEnabled();
```

Comment:
* To better understand this condition: the intended behavior is to not fall back to CPU when symbolic shape are enabled. Correct?

* If yes, to above, are you planning to add other dynamic ops like `unique` and `masked_select` to the list? (realizing these ops may be out of scope for this PR)

Reply:
That is correct, and yes, we will add at least `unique` to the list, I will need to look at `masked_select` some more.